app/src/main/java/com/limelight/binding/input/ControllerHandler.java

```diff
@@ -827,6 +831,22 @@ private int handleRemapping(InputDeviceContext context, KeyEvent event) {
             }
         }
 
+        // Switch A with B and X with Y button for 8BitDo controllers
+        if (context.is8BitDoBtController) {
+            switch (event.getScanCode()) {
+                case 304:
+                    return KeyEvent.KEYCODE_BUTTON_B;
+                case 305:
+                    return KeyEvent.KEYCODE_BUTTON_A;
+                case 306:
+                    return KeyEvent.KEYCODE_HOME;
```

Comment:
Which button on the controller is 306?

If it's an additional center button (in addition to Start and Select), you should use `KEYCODE_BUTTON_MODE` instead of `KEYCODE_HOME`.

Reply:
Yes it is an additional button. I will change the value to `KEYCODE_BUTTON_MODE`